src/test/java/com/virtuslab/using_directives/reporter/ReporterTest.java

```diff
@@ -30,14 +30,19 @@ private void checkDiag(
 
   @Test
   public void reportNotQuotedString() {
-    PersistentReporter reporter = runTest("\n\n  using options -Xfatal-warnings");
+    PersistentReporter reporter = runTest("\n\n  using options Xfatal-warnings");
     assertTrue(reporter.hasErrors());
-    assertEquals(1, reporter.getDiagnostics().size());
+    /* There are two errors reported:
+     - one that reports Xfatal-warnings is not a valid value
+     - one that reports that there's something left in the line
```

Comment:
Why we have `something left` in the line? Also, can we check how `-XXfatal-warnings` would fail? It is quite common case that we need to cover.

Reply:
Because `-XXfatal-warnings` is not a valid value unless it gets wrapped in quotes